WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1625,27 +1623,18 @@ private void performPrimaryAction() {
     }
 
     private void showGutenbergInformativeDialog() {
-        // Show the GB informative dialog on editing GB posts
-        final PromoDialog gbInformativeDialog = new PromoDialog();
-        gbInformativeDialog.initialize(TAG_GB_INFORMATIVE_DIALOG,
-                getString(R.string.dialog_gutenberg_informative_title),
-                mEditPostRepository.isPage() ? getString(R.string.dialog_gutenberg_informative_description_page)
-                        : getString(R.string.dialog_gutenberg_informative_description_post),
-                getString(org.wordpress.android.editor.R.string.dialog_button_ok));
-
-        gbInformativeDialog.show(getSupportFragmentManager(), TAG_GB_INFORMATIVE_DIALOG);
+        // We are no longer showing the dialog, but we are leaving all the surrounding logic because
+        // this is going in shortly before release, and we're going to remove all this logic in the
+        // very near future.
+
         AppPrefs.setGutenbergInfoPopupDisplayed(mSite.getUrl(), true);
```

Comment:
I left this preference because that preference is [used as an early exit in the setGutenbergEnabledIfNeeded function](https://github.com/wordpress-mobile/WordPress-Android/blob/76a49558062397c35be6c6a19e671fd3aba7c479/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L1642-L1644), and I wanted to avoid any changes that would affect the control flow.

Reply:
I don't think `showGutenbergInformativeDialog` is called anywhere anymore, so this code shouldnt be relevant. But I don't want to be too picky since we want to err on the side of being safe and not perfect in stripping out this functionality. We can remove this after ongoing releases to be safer if we prefer.

Reply:
Good catch @cameronvoell ! This means that I actually was changing the control-flow in a small way--exactly the thing I wanted to avoid. For that reason, I have updated this to make fewer changes and leave the control-flow intact. Sorry to ask you to retest this, but I think that's just the safer approach at this point.

In addition, there is an additional step of backgrounding the app that I added to the testing instructions (in bold). It is important to background the app after logging in because it is when the app is backgrounded that this code gets run.

I also wanted to alert you that when I had a site with a default of Aztec and as part of the gutenberg-default-migration the POST call is made to `wpcom/v2/me/gutenberg` to update the mobile editor default to be gutenberg, the response would indicate that the site was now defaulting to gutenberg on mobile, _but on some occasions it would not update and the response would say that the site was still Aztec_. I need to investigate this further, but I wanted to alert you to it in case you see anything like that. I believe this issue is unrelated to any app-side logic, so I don't think it's anything that should block this PR. I'll be investigating it further when I remove the gutenberg toggle.